core/conversion/var/Var.cpp

```diff
@@ -85,6 +85,27 @@ std::string Var::type_name() const {
   }
 }
 
+nvinfer1::ITensor* Var::ITensorOrFreeze(ConversionCtx* ctx, const torch::jit::Node* n) {
+  TRTORCH_CHECK(isITensor() || isIValue(), "Requested either IValue or ITensor, however Var type is " << type_name());
+  nvinfer1::ITensor* out;
+
+  if (isIValue()) {
+    auto weights = converters::Weights(ctx, ptr_.ivalue->toTensor());
+
+    auto const_layer = ctx->net->addConstant(weights.shape, weights.data);
+    TRTORCH_CHECK(const_layer, "Unable to freeze tensor for node: " << *n);
+    const_layer->setName((util::node_info(n) + " [Freeze Tensor]").c_str());
```

Comment:
I am okay dropping this to simplify the function declaration, we could create a random id (a good way is to just cast the pointer to an int) and then print that id and set it as the name so it would be like in the debug messages 
"Freezing tensor as const layer id: 0xDEADBEAF" and the layer name is "0xDEADBEAF [Freeze Tensor]"

Reply:
done